zipkin-web/src/main/scala/com/twitter/zipkin/web/mustache/ZipkinMustache.scala

```diff
@@ -1,39 +1,18 @@
 package com.twitter.zipkin.web.mustache
 
-import com.github.mustachejava.DefaultMustacheFactory
 import java.io._
-import com.twitter.mustache.ScalaObjectHandler
-
-import collection.JavaConversions.mapAsJavaMap
-
-class ZipkinMustache(templateRoot: String, cache: Boolean) {
-  import java.io.Reader
-  class ZipkinMustacheFactory extends DefaultMustacheFactory() {
-    override def getReader(rn: String): Reader = {
-      // hack to get partials to work properly
-      val name = if (rn.startsWith("public")) rn else "templates/" + rn
 
-      if (cache) super.getReader(name) else {
```

Comment:
classpath isn't mutable, so caching is effectively always on, now, as that's the mustache default.